### DISCUSSION TITLE
Registration for the 2024 OpenRadioss Users' Day is now Open, in Philadelphia, Sept. 26th 2024 !

### DISCUSSION THREAD

**Discussion by MarianBulla** [OpenRadioss/OpenRadioss#2718] [Announcements]
Registration for the 2024 OpenRadioss Users' Day is now Open!

The OpenRadioss Community are delighted to be invited to Philadelphia USA by Elham Sahraei,
Associate Professor, Department of Mechanical Engineering, Temple University and its centre of
excellence for the safety of electric vehicle battery packs. Join us for an entire day of
thought-provoking technical presentations and discussions on recent advances made by the
OpenRadioss Communty.
As the number of attendees is limited, register today for this in-person event!

See also:
www.openradioss.org
https://www.linkedin.com/feed/update/urn:li:ugcPost:7204407109436768256/

Thanks and best regards and looking forward meeting you there,
Marian

**Comment by elequiniou** [comment]
Dear community members, 

Please, do not forget to register and actively participate to your community event!
In addition to listen to exciting presentations, we will be happy to give you the opportunity to share your feedbacks and ideas !
Here is the registration link with the updated program: https://web.altair.com/openradioss-users-day-2024

See you soon in Philadelphia!
Eric.
